cluster/src/main/java/com/hartwig/pipeline/report/PipelineResultsProvider.java

```diff
@@ -25,10 +25,14 @@ public static PipelineResultsProvider from(final Storage storage, final Argument
     public PipelineResults get() {
         Bucket reportBucket = storage.get(arguments.patientReportBucket());
         if (reportBucket == null) {
-            reportBucket = storage.create(BucketInfo.newBuilder(arguments.patientReportBucket())
+            BucketInfo.Builder builder = BucketInfo.newBuilder(arguments.patientReportBucket())
                     .setStorageClass(StorageClass.REGIONAL)
-                    .setLocation(arguments.region())
-                    .build());
+                    .setLocation(arguments.region());
+            if (arguments.cmek().trim().isEmpty()) {
```

Comment:
I don't think its actually possible for cmek to be empty, its a mandatory argument now.

Reply:
It can be set to a string of spaces (and maybe the empty string too, I didn't check that) using quotes on the command line.